packages/node/src/config.js

```diff
@@ -30,15 +30,15 @@ module.exports = {
   },
   onUncaughtException: {
     defaultValue: () => (err, report, logger) => {
-      logger.error(`Uncaught exception${getContext(report)}, the process will now terminate…\n${(err && err.stack) ? err.stack : err}`)
+      logger.error(`Uncaught exception${getContext(report)}, the process will now terminate…\n${(err && err.stack) ? err.stack : JSON.stringify(err)}`)
       process.exit(1)
     },
     message: 'should be a function',
     validate: value => typeof value === 'function'
   },
   onUnhandledRejection: {
     defaultValue: () => (err, report, logger) => {
-      logger.error(`Unhandled rejection${getContext(report)}…\n${(err && err.stack) ? err.stack : err}`)
+      logger.error(`Unhandled rejection${getContext(report)}…\n${(err && err.stack) ? err.stack : JSON.stringify(err)}`)
```

Comment:
Expected behaviour (in this specific case):
![image](https://user-images.githubusercontent.com/23224757/61774062-5f10b000-adf6-11e9-8287-708463361604.png)